ipc_plugin/plugin_loader.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3.5m
+#
+#   Copyright (C) 2017, Collabora Ltd.
+#
+#   This Source Code Form is subject to the terms of the Mozilla Public
+#   License, v. 2.0. If a copy of the MPL was not distributed with this file,
+#   You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import importlib
+
+class PluginLoader(object):
```

Comment:
If this class will host the receive/send calls that is fine. You can call them directly. But in that case it should be called something else, like IPC.